Задание 3_9/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Задание_3_9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x для вычисления y ");
+            double x = double.Parse(Console.ReadLine());
+            Console.WriteLine(Metod(x));
+            Console.ReadKey();
+        }
+        static double Metod(double x)
```

Comment:
Уж лучше так
```suggestion
        static double F(double x)
```
или хотя бы так
```suggestion
        static double Method(double x)
```